client/CPlayerInterface.cpp

```diff
@@ -232,7 +230,7 @@ void CPlayerInterface::performAutosave()
 					+ std::to_string(cb->getDate(Date::WEEK))
 					+ std::to_string(cb->getDate(Date::DAY_OF_WEEK));
 
-			cb->save("Saves/Autosave/" + prefix + "Autosave_" + id + "_" + stringifiedDate);
+			cb->save("Saves/Autosave/" + prefix + "Autosave_" + stringifiedDate);

```

Comment:
"Autosave_" seems redundant, since we are in a folder called /Autosave.

Reply:
Yes, what do you suggest? Only counter / day?

Reply:
prefix + date should do the same thing.